src/System.IO.Compression/src/System/IO/Compression/ZipVersion.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.IO.Compression
+{
+    internal enum ZipVersionNeededValues : ushort { Default = 10, ExplicitDirectory = 20, Deflate = 20, Zip64 = 45 }
+
+    /// <summary>
+    /// The upper byte of the "version made by" flag in the central directory header of a zip file represents the 
+    /// OS of the system on which the zip was created. There are up 21 total OS values but we only use the three
```

Comment:
The header tags for Windows and Unix (0,3) are equivalent to those used in ZLib and GZip so I got them from there. Beyond those values I'm not sure - the zip specification doesn't have an RFC/ISO that I could find/access.

Now that I've removed ```Undefined``` as a value, I'm going to rework this comment.